keanu-project/src/main/java/io/improbable/keanu/tensor/Tensor.java

```diff
@@ -40,6 +43,22 @@ static BooleanTensor elementwiseEquals(Tensor a, Tensor b) {
         return new GenericTensor<>(shape);
     }
 
+    static DoubleTensor create(Double value, int[] shape) {
+        return DoubleTensor.create(value, shape);
+    }
+
+    static IntegerTensor create(Integer value, int[] shape) {
+        return IntegerTensor.create(value, shape);
+    }
+
+    static BooleanTensor create(Boolean value, int[] shape) {
+        return BooleanTensor.create(value, shape);
+    }
+
+    static <T> GenericTensor<T> create(T value, int[] shape) {
+        return new GenericTensor<>(value, shape);
```

Comment:
for consistency with the other Tensor types, we should probably also have a `GenericTensor.create` method